components/JoinCommunityButton/JoinCommunityButton.styled.tsx

```diff
@@ -21,3 +22,11 @@ export const StyledJoinCommunityButton = styled(Button)`
     color: ${({ theme }) => theme.palette.primary.main};
   }
 `;
+
+export const StyledCustomLink = styled(CustomLink)`
+  width: 205px;
```

Comment:
fixed width? most of the time its a bad idea

Reply:
On the design the with is fixed